config/environments/production.rb

```diff
@@ -49,5 +49,9 @@
         Sentry.capture_exception(exception)
       },
     }
+
+    cache_store_config.merge({ password: ENV['LAGO_REDIS_CACHE_PASSWORD'] }) if ENV['LAGO_REDIS_CACHE_PASSWORD'].present?
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)